editor/src/clj/editor/camera_editor.clj

```diff
@@ -160,9 +164,9 @@
         near-v2 (gen-outline-vertex (.x near-p2) (.y near-p2) (.z near-p2) (.w near-p2) cr cg cb)
         near-v3 (gen-outline-vertex (.x near-p3) (.y near-p3) (.z near-p3) (.w near-p3) cr cg cb)
 
-        cr-less (* cr 0.75)
-        cg-less (* cg 0.75)
-        cb-less (* cb 0.75)
+        cr-less (* ^double cr 0.55)
+        cg-less (* ^double cg 0.55)
+        cb-less (* ^double cb 0.55)
```

Comment:
I really liked how the part of the frustum that was outside of the clipping volume was dimmed out, but felt it was a bit too subtle, so I dimmed it a bit more.